Model/Config/Backend/EnableProductProtection.php

```diff
@@ -53,23 +53,6 @@ public function afterSave()
         if ($isPPV2Enabled === 1) {
             $attributeSet = $this->attributeSetInstaller->createAttributeSet();
             $this->productInstaller->createProduct($attributeSet);
-        } else {
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_CART_OFFER,
-                0
-            );
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_DISPLAY_PAGE_OFFER,
-                0
-            );
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_POST_PURCHASE_LEAD_MODAL_OFFER,
-                0
-            );
-            $this->writer->save(
-                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_CATALOG_PAGE_MODAL_OFFER,
-                0
-            );
```

Comment:
removing these because they also do not need to be dependent - each offer's `phtml` template has individual rendering logic that will consider the PP toggle at the relevant scope level, so if PP is off, the offers will not display regardless of what the individual display setting is set to

Reply:
after merging in the cart balancing code, i also removed it here. i added an `ifconfig` so cart balancing will only render if the Extend module is enabled - then i also changed the boolean value passed to the normalization function to only be true if a) `normalization` is enabled and b) `product protection` is enabled for the relevant scope